docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -530,6 +530,31 @@ The syntax is as follows:
 Check the [Luck Perms documentation](https://luckperms.net/wiki/Context)
 for an in-depth explanation on what contexts are and how to add them to permissions.
 
+### Permissions
+In case you still want to add and remove permissions directly and dont want to have the chat spam when you use the
+command events, you can use the `luckperms addPermission' and `luckperms removePermission` events. It's designed to
+handle the most important functions of LuckPerms.
```

Comment:
The command event should not spamm the console. So is this maybe a bug?

Reply:
Just a wrong formulation on my part